drivers/gles3/shaders/effects/copy.glsl

```diff
@@ -1,4 +1,6 @@
 /* clang-format off */
+#version 300 es
```

Comment:
This shouldn't be in the .glsl files - the right `#version` will be added by `shader_gles3.cpp` depending on if we are using real GLES or GLES over OpenGL.